### PR TITLE
CHK-1476-added-timestamp-into-completed-event

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Added the PGS completion timestamp to the authorization completion event.

#### Motivation and Context
The change was made to correctly value in closePayment asynchronously the PGS authorization completion timestamp value.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionAuthorizationCompletedData.java [hunk 1]
```diff
@@ -25,6 +27,12 @@ public class TransactionAuthorizationCompletedData {
     @Nullable
     private String rrn;
 
+    /**
+     * PGS authorization completion timestamp
+     */
+    @Nullable
+    private OffsetDateTime timestampOperation;
```
Comment (pietro-tota) [inline src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionAuthorizationCompletedData.java:34]: Change this field to a String just as done for [event timestamp](https://github.com/pagopa/pagopa-ecommerce-commons/blob/e330d285605efdd732bab90e5296a34c3d800d34/src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionEvent.java#L35)(avoiding mongo serialization/deserialization issues) see

**Review by pietro-tota — DISMISSED** [review]

**Review by ciuffagianluca — APPROVED** [review]
LGTM!

**Review by infantesimone — APPROVED** [review]

**Review by MarcoAGreco — DISMISSED** [review]
Nice :)

**Review by pietro-tota — APPROVED** [review]

**Comment by pietro-tota** [comment]
Just some code smells to be fixed. Plus remember to update commons version also in transaction-scheduler-service

**Pull request merged**